src/components/Favourite.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import FavouriteIcon from "../icons/FavouriteIcon";
+
+const WrapperDiv = styled.div`
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-weight: medium;
+  font-size: 20px;
+  border: none;
+  color: #242623;
+  padding: 0px;
+`;
+
+const StyledFavouriteIcon = styled(FavouriteIcon)`
+  cursor: pointer;
+  filter: drop-shadow(2px 2px 2px #00000029);
```

Comment:
`#00000029` looks a little strange. I'd also recommend using `rgba()` for a transparent shadow

Reply:
Copied it from XD's developer link... 🤷‍♀️